client/signup/steps/design-type-with-store/index.jsx

```diff
@@ -121,11 +118,11 @@ export default React.createClass( {
 			default:
 				return <PressableStoreStep
 							{ ... this.props }
-							ref={ this.onPressableStoreStepRef }
+							ref={ ( ref ) => this.pressableStore = ref }
```

Comment:
Do only working solution that I was able to come up with was to pass that function to child component as prop, and call it in `componentDidMount`.